hugegraph-dist/pom.xml

```diff
@@ -181,6 +181,7 @@
                         <configuration>
                             <tasks>
                                 <echo file="${project.basedir}/dist.sh">
+                                    wget --version 1>/dev/null || exit
```

Comment:
**TODO:** use curl to replace it (and refer a better way to handle it -- like maven property)